src/main/java/rs/teslaris/core/dto/document/ConferenceBasicAdditionDTO.java

```diff
@@ -0,0 +1,29 @@
+package rs.teslaris.core.dto.document;
+
+import java.time.LocalDate;
+import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import rs.teslaris.core.dto.commontypes.MultilingualContentDTO;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ConferenceBasicAdditionDTO {
+
+    private Integer id;
+
+    @Valid
+    @NotNull(message = "You have to provide conference name.")
+    private List<MultilingualContentDTO> name;
+
+    @NotNull(message = "You have to provide at least a year when this conference took place.")
```

Comment:
Even if exact date is catalogued for a conference, it should be enabled to find that by using only year. I mean if we have in database:

CRIS conference, 15/5/2024-17/5/2024, Vienna, Austria

we can find that by typing CRIS 2024

Reply:
Ok, I can add dates to index